includes/wc-payment-api/class-wc-payments-api-client.php

```diff
@@ -1919,10 +1920,23 @@ protected function check_response_for_errors( $response ) {
 					$response_code
 				);
 			} elseif ( isset( $response_body['error'] ) ) {
+				$response_body_error_code = $response_body['error']['code'] ?? null;
+				$payment_intent_status    = $response_body['error']['payment_intent']['status'] ?? null;
+
+				// Check for an 'amount_too_large' error code and a 'requires_capture' payment intent status.
```

Comment:
We can skip this specific line - if-condition stating the same =)

Reply:
Removed here  https://github.com/Automattic/woocommerce-payments/pull/7581/commits/3d4250aa6e97c15a36e66d7eb8b5eddfd19d6504 thanks!